users/dierkes/preprocessing/segment_cutting.py

```diff
@@ -0,0 +1,85 @@
+import soundfile as sf
+import numpy as np
+from i6_core.lib import corpus
+import multiprocessing
+import tqdm
+
+from sisyphus import Job, Task, tk
+
+class CutSpeechSegmentsFromAudio(Job):
+    """
+    This Job uses a transcription file to cut all segments containing speech from the audio files
+    and stitches them together to files of a target length. Segments that are
+    longer than the target_length are stored as individual files.
+
+
+    """
+    def __init__(self, bliss_corpus, target_length=20, n_workers=16, file_extension='.wav'):
+        """
+
+        :param tk.Path bliss_corpus:
+        :param int target_length: target length of the stitched extracted speech in seconds
+        :param int n_workers: number of threads for work on files in parallel
+        :param str file_extension: currently supported are wav, flac and mp3
+        """
+        self.bliss_corpus = bliss_corpus
+        self.target_length = target_length
+        self.n_workers = n_workers
+        self.file_extension = file_extension
+        assert file_extension in ['.wav', '.flac', '.mp3']
```

Comment:
From other jobs, it seems like the convention would be to rather use `"wav"` instead of `".wav"`, see e.g. https://github.com/rwth-i6/i6_core/blob/main/corpus/transform.py